applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -369,6 +369,61 @@ config BT_AUDIO_PACKING_INTERLEAVED
 	help
 	  ISO channels can either be interleaved or sequentially packed; sequential is the default one.
 
+config BT_AUDIO_PREFERRED_MIN_PRES_DLY_US
+	int "The preferred minimum presentation delay"
+	range AUDIO_MIN_PRES_DLY_US AUDIO_MAX_PRES_DLY_US
+	default 10000
+	help
+	  The preferred minimum presentation delay in micro seconds. This can not
+	  be less than the absolute minimum presentation delay.
+
+config BT_AUDIO_PREFERRED_MAX_PRES_DLY_US
+	int "The preferred maximum presentation delay"
+	range AUDIO_MIN_PRES_DLY_US AUDIO_MAX_PRES_DLY_US
```

Like Erik suggested: 
```suggestion
	range BT_AUDIO_PREFERRED_MIN_PRES_DLY_US AUDIO_MAX_PRES_DLY_US
```